angular/Tile/Overview.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Overview
-description: overview
+title: Overview in Angular Tile Control | Syncfusion
```

Comment:
Overview of Angular Tile Control | Syncfusion

Reply:
changed